website/versioned_docs/version-0.67/integration-with-android-fragment.md

```diff
@@ -101,12 +173,51 @@ Now in your Activity class e.g. `MainActivity.java` you need to add an OnClickLi
 
 Add the button field to the top of your Activity:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+private lateinit var button: Button
+```
+
+</TabItem>
+<TabItem value="java">
+
 ```java
 private Button mButton;
 ```
 
+</TabItem>
+</Tabs>
+
 Update your Activity's onCreate method as follows:
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle): Unit {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.main_activity)
+
+    button = findViewById(R.id.button) as Button
```

Comment:
I am pretty sure Android has a generic option now?
```suggestion
    button = findViewById<Button>(R.id.button)
```

You probably don't even need the `<>` specification, but not sure.